src/main/java/step2/domain/ScoreType.java

```diff
@@ -0,0 +1,57 @@
+package step2.domain;
+
+import java.util.Arrays;
+
+public enum ScoreType {
+    NONE(0, "", 0, false),
+    THREE(5000, " (5000원)", 3, false),
+    FOUR(50000, " (50000원)", 4, false),
+    FIVE(1500000, " (1500000원)", 5, false),
+    FIVE_BONUS(30000000, ", 보너스 볼 일치(30000000원)", 5, true),
+    SIX(2000000000, " (2000000000원)", 6, false);
+
+    private int prize;
+    private String WinnerMsg;
+    private int targetNumber;
+    private boolean hasBonus;
+
+    ScoreType(int prize, String WinnerMsg, int targetNumber, boolean hasBonus) {
+        this.prize = prize;
+        this.WinnerMsg = WinnerMsg;
+        this.targetNumber = targetNumber;
+        this.hasBonus = hasBonus;
+    }
+
+    public static ScoreType getScore(int inputNumber) {
+        return Arrays.asList(values())
+                .stream()
```

Comment:
```suggestion
        return Arrays.stream(values())
```

위와 같이 바로 Arrays.stream을 호출하실수도 있을 것 같아요